service/controller/resource/tccpn/template/template_main_etcd_volume.go

```diff
@@ -16,6 +16,12 @@ const TemplateMainEtcdVolume = `
       - Key: Name
         Value: {{ .Name }}
       VolumeType: gp3
+      {{- if and (ge .Iops 3000) (le .Iops 16000) }}
```

Comment:
how are those bounds defined? do they change by disk size and or instance type?

Reply:
those bounds are set by volume type. according to the docs we are bound to those numbers if we use gp3, see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-blockdevicemapping-ebs.html